trillian/ctfe/ct_server/main.go

```diff
@@ -122,8 +121,12 @@ func main() {
 			etcdRes.Update(ctx, *etcdMetricsService, byeMetrics)
 		}()
 	} else {
-		// Use a fixed endpoint resolution that just returns the addresses configured on the command line.
-		res = util.FixedBackendResolver{}
+		// Use endpoint resolution that just returns the addresses configured on the command line.
+		r, err := naming.NewDNSResolverWithFreq(time.Second)
+		if err != nil {
+			glog.Fatalf("Failed to create new DNSResolver: %v", err)
```

Comment:
`glog.Exitf`  is probably better here (no need for a stack trace)

Reply:
Good call, done.